torch/onnx/utils.py

```diff
@@ -805,7 +817,7 @@ def set_names(node_list, name_list, descriptor):
     set_names(list(graph.inputs()), input_names, "input")
     set_names(list(graph.outputs()), output_names, "output")
 
-attr_pattern = re.compile("^(.+)_([ifstgz])$")
```

Comment:
Just curious, what‘s the meaning of `z` at this regular expression? It never appeared in our PyTorch symbolic function files.